recipes-core/images/core-image-tedge.bb

```diff
@@ -1,14 +1,7 @@
-require recipes-core/images/core-image-minimal.bb
+require recipes-core/images/core-image-base.bb
 
 IMAGE_INSTALL:append = " \
-    thin-edge \
     tedge \
-    tedge-mapper \
-    tedge-agent \
-    c8y-configuration-plugin \
-    c8y-firmware-plugin \
-    c8y-log-plugin \
-    c8y-remote-access-plugin \
-    tedge-apt-plugin \
-    tedge-watchdog \
+    tedge-bootstrap \
```

Comment:
question: if `core-image-tedge` requires `tedge-bootstrap`, which requires systemd, is it even possible to run with sysvinit?

Reply:
It is getting harder and harder to support sysvinit as more and more features need systemd to work...so I don't mind   looking at sysvinit support in another PR.

Reply:
Yeah this recipe should be attached under condition that systemd is present. I misunderstood `REQUIRED_DISTRO_FEATURES` variable in yocto and I thought it will do it automatically.

Reply:
This can be fixed in https://github.com/thin-edge/meta-tedge/issues/65